tests/behat/features/gridfield-navigation.feature

```diff
@@ -0,0 +1,160 @@
+Feature: Search in GridField
+  As an author
+  I want to navigate through the filtered and sorted GridField list
+  So that I see the filtered and sorted GridField when I navigate to an item and back to the GridFIeld list  
+
+  Background:
+    Given the "Company" "Walmart" with "Category"="Retail"
+      And the "Employee" "Alen" with "Company"="1"
+      And the "Employee" "Bill" with "Company"="1"
+      And the "Employee" "Ford" with "Company"="1"
+      And the "Company" "ExxonMobil" with "Category"="Oil"
+      And the "Company" "Vitol" with "Category"="Other"
+      And the "Company" "Safeway" with "Category"="Other" 
+      And the "Company" "SuperValue" with "Category"="Other" 
+      And the "Company" "UBS" with "Category"="Other"
+      And the "Company" "Kraft Foods" with "Category"="Other" 
+      And the "Company" "Ahold" with "Category"="Other"
+      And the "Company" "Cisco" with "Category"="Other"
+      And the "Company" "Bouygues" with "Category"="Other" 
+      And the "Company" "Insurance" with "Category"="Other"
+      And I am logged in with "ADMIN" permissions
+
+  Scenario: I can navigate back through the GridField items by using better button
+    When I go to "/admin/gridfield-test-navigation"
+      And I should see "Walmart" in the "#Form_EditForm" element
+      And I should see "ExxonMobil" in the "#Form_EditForm" element
+      And I should see "Vitol" in the "#Form_EditForm" element
+      And I should see "Safeway" in the "#Form_EditForm" element
+      And I should see "SuperValue" in the "#Form_EditForm" element
+      But I should not see "UBS" in the ".col-Name" element
+      And I should see "3" in the ".pagination-page-number" element
```

Comment:
Why are you explicitly testing for the number "3"? Wouldn't we start on page 1?

Reply:
We have 3 pages and I check that number of pages is 3. We start on page 2.

Reply:
Initially we are on page 1 and check that we have 3 pages with GridField. After that we click button "Next" in pagination and move to page number 2. `Then I click on the "#action_pagination_next" element` 
`Then I click on the ".action--previous" element`  to come back to page 1.